testsuite/tests/with/dynamic-dependencies/test.py

```diff
@@ -0,0 +1,57 @@
+"""
+Check that dynamic dependencies don't affect the use of `alr with`
```

Comment:
Looks like there is twice the same test.

Reply:
Forgot to save :sweat_smile:. Should be fixed as of 1a3b6d8.